features/command_line/bisect.feature

```diff
@@ -106,8 +107,10 @@ Feature: Bisect
           - ./spec/calculator_7_spec.rb[1:1]
           - ./spec/calculator_8_spec.rb[1:1]
           - ./spec/calculator_9_spec.rb[1:1]
+      Checking that failures are order-dependent..
+       - Running: rspec ./spec/calculator_1_spec.rb[1:1] --seed 1234 (n.nnnn seconds) failure is order-dependent
```

Comment:
The "failure is order dependent" bit isn't separated at all from the command so it kinda runs together.  Thoughts on adding an elipses or something similar to separate?